resume.md

```diff
@@ -1,49 +1,48 @@
-# FIRST LAST NAME
+# Daniel Qian
 
-(123) 456 - 7890 | email@gmail.com
+(925) 922 - 4281 | danielq3@illinois.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/DAQIANN | linkedin.com/danielq5/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of Illinois Urbana Champaign** *May 2023* *GPA 3.89* *Dean's List*
+* Bachelor of Science in Mathematics and Computer Science 
+* Minor in Statistics 
+* **Coursework**: Data Structures, Numerical Methods, Software Design Studio, Discrete Structures, Linear Algebra, Statistical Analysis, Statistics and Probability
+
+## EXPERIENCE
+**Course Assistant | UIUC | Champaign, IL  |September 2020 – Present**
+* Founding member of CS 128 (Introduction to Computer Science II), a new core Computer Science class to be implemented Spring 2021
```

Comment:
Cool that it's a new course, what is the need that you're solving?  Why is this important?  Why didn't this course exist previously, how did you push for this new course?  What is going to be taught in this new course?